pkgs/applications/audio/virtual-ans/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchurl
+, unzip
+, xorg
+, libGL
+, alsaLib
+, SDL2
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "virtual-ans-${version}";
+  version = "3.0.2c";
+
+  src = fetchurl {
+    url = "https://warmplace.ru/soft/ans/virtual_ans-${version}.zip";
+    sha256 = "1dd4d91yjmp7xfjcjlam8l55r4963ypjgll97n4gjvl72m1fq856";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Photoelectronic microtonal/spectral musical instrument";
+    homepage = "https://warmplace.ru/soft/ans/";
+    license = licenses.free;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ jacg ];
+  };
+
+  nativeBuildInputs = [
```

Comment:
A lot of these should be `buildInputs` instead of `nativeBuildInputs`, i think.

Reply:
... all of them seem to work as `buildInuputs`.

Reply:
As described in the [nixpkgs manual](https://nixos.org/nixpkgs/manual/#ssec-stdenv-dependencies), some should still be `nativeBuildInputs`. In this case, just `autoPatchelfHook` i think.

Reply:
I really struggle to find the information I need about details of how things work in nix. Thanks for all the links. This one in particular I'll have to read carefully when I have some time.